src/main/java/duke/commands/FindCommand.java

```diff
@@ -0,0 +1,36 @@
+package duke.commands;
+
+import java.util.ArrayList;
+
+import duke.data.TaskList;
+import duke.storage.Storage;
+import duke.ui.Ui;
+
+/**
+ * Finds and lists events with the keywords provided by the user.
+ */
+public class FindCommand extends Command {
+    private final String searchString;
+
+    public FindCommand(String userInput) {
+        this.searchString = userInput.substring(5);
+    }
+
+    @Override
```

Comment:
shouldn't we also write javadocs for subclass methods even if overriden? I think there's a intellij function for this, you can search it up on google.